Web3/Classes/Core/Transaction/EthereumTransaction.swift

```diff
@@ -306,7 +366,7 @@ extension EthereumSignedTransaction: RLPItemConvertible {
             v: EthereumQuantity(quantity: v),
             r: EthereumQuantity(quantity: r),
             s: EthereumQuantity(quantity: s),
-            chainId: 0
+            chain: .ethereumMain
```

Comment:
Ok so I know we can't infer this value from a signature. But I'm not confident that mainnet is the right choice here. Actually there is no real right choice. Maybe we should drop official support for `RLPItemInitializable`...

What do you think @pixelmatrix

Reply:
I had no idea what this initializer was, so I just adapted it, but yea I think there shouldn't be a default value :s 
Let me know what can be done here gentleman :)

Reply:
I think what @Ybrin is suggesting here is to make this extension just conform to RLPItemRepresentable instead of RLPItemConvertible. Makes sense to me, as long as there aren't parts of the library that depend on it.